backend/dal/Services/Admin/Concrete/UserService.cs

```diff
@@ -36,11 +36,25 @@ public class UserService : BaseService<User>, IUserService
         /// <param name="quantity"></param>
         /// <param name="sort"></param>
         /// <returns></returns>
-        public Paged<User> GetNoTracking(int page = 1, int quantity = 10, string sort = null)
+        public Paged<User> GetNoTracking(int page = 1, int quantity = 10, string sort = null, Guid? userId = null)
```

Comment:
I'd remove the `userId` filter argument.  It makes more sense for a specific use-case (i.e. GetMyUsers(...) or GetAgencyUsers(...)).

If we do want to make a generic `GetNoTracking(...)` function then we need to pass filter parameters to it as an filter object or a dictionary.